README.md

```diff
@@ -1,12 +1,15 @@
 # News Site
-
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+A news site built with HTML and CSS
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Everything works but when I try to push it to netlify I get this code: 
+Page Not Found
+Looks like you've followed a broken link or entered a URL that doesn't exist on this site.
```

Comment:
You need to specify publish folder as the `/code` folder. If you add "/code" after your URL it works (although it's of course better to specify the publish folder instead :) )